fw/http.c

```diff
@@ -981,6 +977,16 @@ tfw_http_resp_pair_free(TfwHttpReq *req)
 	tfw_http_conn_msg_free((TfwHttpMsg *)req);
 }
 
+void
+tfw_http_resp_pair_free_and_put_conn(TfwHttpResp *resp)
```

Comment:
Is it really needed for one-time use? Why not add a `put()` call where it's called?